cluster/client.py

```diff
@@ -176,7 +173,7 @@ def cluster_deploy(args):
             args.branch,
             master=args.master,
             slave=args.slave,
-            wait=args.wait,
+            no_wait=args.no_wait,
```

Comment:
This is a detail that I won't blok, in general while speaking as well I dislike double negation so to avoid that in your code you cloud let wait=not args.no_wait which makes make brain smarter later in the code.

Also the diff would be probably smaller !

Reply:
yes good idea, I've kept in mind too much the accuracy between options and code